functional-tests/envs/testenv.py

```diff
@@ -156,6 +157,51 @@ def withdraw(
 
         return l2_tx_hash, tx_receipt, total_gas_used
 
+    def withdraw_op_return(
+        self,
+        ctx: flexitest.RunContext,
+        el_address: str,
+        payload: str,
+    ):
```

Comment:
Is it possible to avoid duplicating a bunch of this logic with the standard `withdraw` function by consolidating some of it into a more general method?

Reply:
Yes, I intend to do this with some arbitrary `descriptor` as a single `withdraw` function. Will refactor.

Reply:
Done in b0eef7d99e0f88b31a1a5387f8008991efc5a94e